include/deal.II/base/array_view.h

```diff
@@ -1134,6 +1134,21 @@ make_array_view(const Table<2, ElementType> &                   table,
 
 
 
+/*
+ * Create a view that doesn't modify the container it points to.
+ * This is useful if the object passed in is not `const`.
+ *
+ * @relatesalso ArrayView
+ */
+template <typename Container>
+inline auto
+make_const_array_view(const Container &container)
+  -> decltype(make_array_view(container))
+{
+  return make_array_view(container);
```

Comment:
Hm, so what is the difference between this function and `make_array_view`?

Reply:
It makes sure that the `container` is `const`. For `std::vector` there is
```
ArrayView<ElementType> make_array_view (std::vector<ElementType>& vector)  
```
and
```
ArrayView<const ElementType> make_array_view (const std::vector<ElementType>& vector)  
```
. `make_const_array_view` would return `ArrayView<const ElementType>` in both cases.